node/src/GlideClusterClient.ts

```diff
@@ -567,4 +568,35 @@ export class GlideClusterClient extends BaseClient {
     public time(route?: Routes): Promise<ClusterResponse<[string, string]>> {
         return this.createWritePromise(createTime(), toProtobufRoute(route));
     }
+
+    /**
+     * Displays a piece of generative computer art and the Redis version.
+     *
+     * See https://valkey.io/commands/lolwut/ for more details.
+     *
+     * @param version - An optional argument that can be used to specify the version of computer art to generate.
+     * @param parameters - An optional argument that can be used to specify the output:
+     *  For version `5`, those are length of the line, number of squares per row, and number of squares per column.
+     *  For version `6`, those are number of columns and number of lines.
+     * @param route - The command will be routed to a random node, unless `route` is provided, in which
+     *  case the client will route the command to the nodes defined by `route`.
+     *
+     * @returns A piece of generative computer art along with the current server version.
+     *
+     * @example
+     * ```typescript
+     * const response = await client.lolwut(6, [40, 20], `allNodes`);
+     * console.log(response); // Output: "Redis ver. 7.2.3" - Indicates the current server version.
```

Comment:
From my testing, running with Valkey-cli, it sill returns with "Redis ver. 7.2.5". For all nodes, it will only have one single response.

```
127.0.0.1:6379> lolwut 6, [40,20], "AllNodes"
Redis ver. 7.2.5
```